src/assets/image/icons/Security.svg

```diff
@@ -1 +1,3 @@
-<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512"><!--!Font Awesome Free 6.5.2 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free Copyright 2024 Fonticons, Inc.--><path d="M256 0c4.6 0 9.2 1 13.4 2.9L457.7 82.8c22 9.3 38.4 31 38.3 57.2c-.5 99.2-41.3 280.7-213.6 363.2c-16.7 8-36.1 8-52.8 0C57.3 420.7 16.5 239.2 16 140c-.1-26.2 16.3-47.9 38.3-57.2L242.7 2.9C246.8 1 251.4 0 256 0zm0 66.8V444.8C394 378 431.1 230.1 432 141.4L256 66.8l0 0z"/></svg>
\ No newline at end of file
+<svg width="12" height="16" viewBox="0 0 12 16" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path d="M11.5 4.10084V9.17899C11.5 9.91993 11.3291 10.6114 11.0072 11.2165C10.6848 11.8225 10.2134 12.3376 9.61287 12.7255L6.17668 14.9458C6.0745 15.0145 5.93945 15.0205 5.82903 14.949L2.38713 12.7255C1.78599 12.3376 1.31506 11.8225 0.992775 11.2165C0.67073 10.6114 0.5 9.92006 0.5 9.17899V4.10084L0.501213 4.08477C0.509586 3.89565 0.662478 3.74912 0.84243 3.7583C0.849347 3.75881 3.27864 3.90891 5.76083 1.10929L5.77746 1.09182C5.90936 0.96289 6.11588 0.970669 6.23868 1.10929C8.58886 3.76021 10.891 3.76659 11.1356 3.75919C11.1479 3.75779 11.1601 3.7569 11.1727 3.7569C11.3535 3.7569 11.5 3.91069 11.5 4.10084Z" stroke-width="0.975177" stroke-linecap="round"/>
```

Comment:
Please add a reference where the SVG has been taken from

Reply:
added